server/app/rest_api/rest.py

```diff
@@ -158,12 +159,23 @@ def get(self):
                         }
                     }
                 }
-            }
+            },
+            "400": {
+                "description": "Malformed filter"
+            },
+            "403": {
+                "description": "Non-interactive request"
+            },
         }
     })
     def put(self):
```

Comment:
Ok, yes totally agree. But... I think that this error is outside the scope of this PR. I do a separate PR for that. 

Essentially this was an inconsistency introduced way earlier and I'd like to have a specific commit to point to when it is fixed rather than one muddled by other "central theses" (as Tony would say)

Reply:
Sounds good to me.  Lets just not forget.

Reply:
Working on it now